content/blog/2020-10-02-glosario-sprint/index.markdown

```diff
@@ -0,0 +1,44 @@
+---
+title: Glosario Sprint
+author:
+  - greg
+date: '2020-10-02'
+categories:
+  - learn
+  - package
+tags:
+  - glosario
+description: |
+  This October, help us expand an open source, multilingual glossary of data science terms.
+slug: glosario-sprint
+photo:
+  url: https://unsplash.com/photos/1UDjq8s8cy0
+  author: Joshua Hoehne
+---
+
+In July we announced [Glosario][glosario],
+an open source, multilingual glossary of data science terms that now contains more than 500 terms in multiple languages.
+We hope it will help lesson developers and maintainers make their materials more findable
+as well as helping learners directly.
+
+This month,
+as part of [Hacktoberfest][hacktoberfest],
+we're encouraging all members of our community to help expand Glosario.
+So far, it includes definitions in English, French, Portuguese, and Spanish,
+but it has been designed to include definitions in any language.
+We'd love to see our international community come together to provide definitions and translations
+to increase the value of this resource for everyone.
+
+You don't need to know any particular programming language to contribute to Glosario:
+anyone who is familiar with the GitHub web interface can get involved.
+There is a detailed contributors' guide [on the site][glosario],
```

Comment:
https://carpentries.github.io/glosario/ doesn't include the contributors guide. better link to the GitHub repo from here.

Reply:
Thanks @tobyhodges - fixed.